client/my-sites/customer-home/style.scss

```diff
@@ -83,7 +79,6 @@
 }
 
 .customer-home__section-heading {
-	font-family: 'Recoleta', serif;
```

Comment:
Note that we could also extend `.wp-brand-font` here rather than adding the `wp-brand-font` class to all the components:
```css
.customer-home__section-heading {
  @extend .wp-brand-font;
}
```

Both approaches are fine to me, so feel free to keep the explicit `wp-brand-font` classes if you prefer.

Reply:
I was wondering if the language selectors ('[lang*=en] .wp-brand-font' for example) would still apply if we extended the brand font that way, since `.wp-brand-font` itself doesn't include the language piece, only the fallback font...I'll double-check and see how the SCSS would compile, because that might be an easier method for some areas, for sure.

Reply:
Ooh, that does appear to work as intended! I know `@extend` won't work within media queries, but I don't think we define the font within any media queries. Thanks for nudging me to experiment!